src/chapters/booleans-and-conditionals/exercises.rst

```diff
@@ -0,0 +1,140 @@
+Write variables for these values.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Variable
+     - Value
+   * - engineIndicatorLight
+     - red blinking
+   * - spaceSuitsOn
+     - true
+   * - shuttleCabinReady
+     - true
+   * - crewStatus
+     - spaceSuitsOn && shuttleCabinReady
+   * - computerStatusCode
+     - 200
+   * - shuttleSpeed
+     - 15000
+
+Read the below code. What will be printed to the console?
+----------------------------------------------------------------------
+Answer using the value of ``engineIndicatorLight`` that is defined in the table above.
+
+.. code-block:: javascript
+
+   if (engineIndicatorLight === "green") {
+      console.log("engines have started");
+   } else if (engineIndicatorLight === "green blinking") {
+      console.log("engines are preparing to start");
+   } else {
+      console.log("engines are off");
+   }
+
+Write conditional expressions to satisfy the safety rules below
+----------------------------------------------------------------
+Using the variables defined from the table above.
+
+#. ``crewStatus``
+
+   - if crewStatus is ``true``, print ``"Crew Ready"``
+   - else print ``"Crew Not Ready"``
+
+#. ``computerStatusCode``
+
+   - if value is ``200``, print ``"Please stand by. Computer is rebooting"``
+   - else if value is ``400``, print ``"Success! Computer working fine"``
+   - else print ``"ALERT: Computer not functioning"``
+
+#. ``shuttleSpeed``
+
+   - if value is ``> 17,500``, print ``"ALERT: could escape orbit"``
+   - else if value is ``< 8000``, print ``"ALERT: could fall out of orbit"``
+   - else print ``"Stable speed"``
+
+Do these code blocks produce the same result?
+---------------------------------------------
+Answer Yes or No
+
+.. code-block:: javascript
+
+    if (crewStatus && computerStatusCode === 200 && spaceSuitsOn) {
+        console.log("all systems go");
+    } else {
+        console.log("WARNING. Not ready");
+    }
+
+.. code-block:: javascript
+
+    if (!crewStatus || computerStatusCode !== 200 || !spaceSuitsOn) {
+        console.log("WARNING. Not ready");        
+    } else {
+        console.log("all systems go");
+    }
+
+Implement logic for spectrometer
+--------------------------------
+A spectrometer is used to identify what objects/planets/gases are physically made of.
+
+| Implement the below rules using ``if``, ``else if``, and ``else`` statements.
+
+* variables defined by spectrometer
+
+  * ``colorCode``
+  * ``waveLength``
+
+* if ``colorCode`` is ``"2A109"`` OR ``waveLength`` is ``2985`` nanometers.
+
+  * print ``"Water found"``
+
+* else if ``colorCode`` is ``"482C0"`` AND ``waveLength`` is ``589.0`` OR ``589.6`` nanometers.
+
+  * print ``"Sodium found"``
+
+* else if ``colorCode`` is ``"Z491"`` OR ``waveLength`` is ``656`` OR ``486`` OR ``434`` nanometers.
+
+  * print ``"Hydrogen found"``
+
+* else print ``"Non target element found"``
+
+Fix the fuel status system
```

Comment:
I would remove the debugging element from this problem, have people code this series of statements, and remove the above problem.

Reply:
I like the idea of having the students evaluate someone else's code.

Maybe we could have the students compare two code blocks.  The first will begin with the comparison to 20000 and proceed down through 5000 (then the ``else`` clause), while the second block begins with the comparison to 5000 and works its way up to 20000 (then ``else``).

This will allow the students to consider how the order of the if statements affects the behavior of the code.  ("IF we want this behavior, HOW should we structure the code?")

If we want the students to code, we could follow the evaluation above with a task asking them to structure if statements that DO begin with 5000 yet still produce the desired output (e.g. the students could replace the ``>`` signs with ``<`` signs).